tests/bgp/test_bgp_azng_migration.py

```diff
@@ -155,11 +200,43 @@ def test_bgp_azng_migration(duthosts, enum_upstream_dut_hostname):
             assert routes_json['filteredPrefixCounter'] == 0
```

Comment:
There is an assert without failed reason too, can you just search "assert " in the script to make sure you didn't miss any places else.